src/Select.js

```diff
@@ -174,11 +184,26 @@ Select.STATE = {
 	},
 
 	/**
+	 * Block or unblock the component behaviours.
+	 * @type {?boolean}
```

Comment:
Since you're setting a validator, it won't accept `null` or `undefined` values, so this should be `{boolean}`, not `{?boolean}`.